src/utils/footer-utils.ts

```diff
@@ -31,7 +31,11 @@ export const shouldOverwriteFooter = (repositoriesOBJ: Repository, originalConte
   const hasCustomFooter = checkForExistingFooter(originalContent);
 
   if (!overwriteExistingFooter && hasCustomFooter) {
-    const answer = promptSync(`metatavu-custom-footer exists already! ${chalk.red("Do you want to overwrite the existing footer for")} ${chalk.magenta(`${owner}/${repoName}`)} (y/N): `);
+    const answer = promptSync(`metatavu-custom-footer exists already! ${chalk.red("Do you want to overwrite the existing footer for")} ${chalk.magenta(`${owner}/${repoName}`)} (y/N/q): `);
```

Comment:
Change the upper case N to lower case n- be consistent I think in one of the instructions it's lowercase